srcpkgs/noisetorch/template

```diff
@@ -0,0 +1,24 @@
+# Template file for 'noisetorch'
+pkgname=noisetorch
+version=0.11.3
+revision=1
+archs="x86_64*"
```

Comment:
Even though this will include musl archs, why limit archs at all? Does the build fail otherwise? If so, is it fixable? It's much preferable to fix cross build than give up on it.

Reply:
I will try to build it through musl later. but I cannot confirm that it builds on other architectures. How should I go about this?

Reply:
Try building it in a musl masterdir or cross compile for example to aarch64-musl. Of course you will have to remove this line.

Reply:
Please remove the restriction so we can see build results on CI.